games-simulation/openrct2/openrct2-0.2.4.ebuild

```diff
@@ -0,0 +1,182 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils xdg-utils
+
+MY_PN="OpenRCT2"
+MY_PN_OBJ="objects"
+MY_PN_RPL="replays"
+MY_PN_TS="title-sequences"
+MY_PV_OBJ="1.0.12"
+MY_PV_RPL="0.0.4"
+MY_PV_TS="0.1.2c"
+
+DESCRIPTION="An open source re-implementation of Chris Sawyer's RollerCoaster Tycoon 2"
+HOMEPAGE="https://openrct2.org/"
+SRC_URI="
+	https://github.com/${MY_PN}/${MY_PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz
+	https://github.com/${MY_PN}/${MY_PN_OBJ}/releases/download/v${MY_PV_OBJ}/${MY_PN_OBJ}.zip -> ${PN}-${MY_PN_OBJ}-${MY_PV_OBJ}.zip
+	https://github.com/${MY_PN}/${MY_PN_TS}/releases/download/v${MY_PV_TS}/${MY_PN_TS}.zip -> ${PN}-${MY_PN_TS}-${MY_PV_TS}.zip
+	test? ( https://github.com/${MY_PN}/${MY_PN_RPL}/releases/download/v${MY_PV_RPL}/${MY_PN_RPL}.zip -> ${PN}-${MY_PN_RPL}-${MY_PV_RPL}.zip )
+"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="dedicated libressl +lightfx +multiplayer +opengl test +truetype twitch"
+REQUIRED_USE="dedicated? ( !lightfx multiplayer !opengl !truetype !twitch )"
+
+DEPEND_RDEPEND="
+	dev-libs/icu:=
+	dev-libs/jansson:=
+	dev-libs/libzip:0=
+	media-libs/libpng:0=
+	sys-libs/zlib:=
+	!dedicated? (
+		media-libs/libsdl2:=
+		media-libs/speexdsp:=
+		opengl? ( virtual/opengl )
+		truetype? (
+			media-libs/fontconfig:=
+			media-libs/freetype:2=
+			media-libs/sdl2-ttf:=
+		)
+		twitch? (
+			net-misc/curl:=[ssl]
+			libressl? ( dev-libs/libressl:0= )
+			!libressl? ( dev-libs/openssl:0= )
+		)
+	)
+	multiplayer? (
+		libressl? ( dev-libs/libressl:0= )
+		!libressl? ( dev-libs/openssl:0= )
+	)
+"
+
+RDEPEND="
+	${DEPEND_RDEPEND}
+	dedicated? (
+		acct-group/openrct2
+		acct-user/openrct2
+	)
+"
+
+DEPEND="
+	${DEPEND_RDEPEND}
+	app-arch/unzip
```

Comment:
Pretty simple: During my proxy-maint work, multiple devs were always asking me to add `app-arch/unzip` into `DEPEND`, when I have `*.zip` to unpack in SRC_URI. So is this information wrong and it should be removed?

Reply:
> So is this information wrong and it should be removed?

I need you to tell me that, you are the candidate here ;) When you have commits access, there will not be people telling you what to do anymore, we will rely on you to get things right. I want you to tell me where this comes from (hint: PMS) and explain if it needs to be removed or tweaked.

Reply:
Despite of my changes, I will answer, when I have spoken with my mentor. I think, that the current change should be correct.

My findings in PMS:
First, PMS definies in `12.3.15`, that for *.zip files, the ebuild must ensure, that `Info-ZIP Unzip` is installed.

But PMS also definies in `8.2`, that listed dependencies in `DEPEND` must not executed their install binaries. But this is allowed in `BDEPEND`, so `app-arch/unzip` should go into that, as unzip must be executed in order to unpack files from *.zip.

Reply:
After speaking with my mentor and reading PMS.

First, PMS definies in `12.3.15`, that for *.zip files, the ebuild must ensure, that `Info-ZIP Unzip` is installed.

Secondly, PMS states, that every build dependency on the host is needed to go into `BDEPEND`. Since the host must unpack the archive, such dependencies must go into `BDEPEND`.

PMS also definies in `8.2`, that listed dependencies in `DEPEND` must not executed their install binaries. But this is allowed in `BDEPEND`, so `app-arch/unzip` should go into that, as unzip must be executed in order to unpack files from *.zip.

But as PMS states, in theory, I would need to do this for every unpack format. Gentoo has the `@system` set, which provides several packages, which are needed to run a Gentoo. Every user has these packages. You can show them with `emerge --pretend @system`, but the definition comes from the profiles. The global setting, which affects every profile is `${PORTDIR}/profiles/base/packages` and optionally arch-specific `./arch/<arch>/packages.

Since `app-misc/unzip` is not listed there, it must be added into `BDEPEND`.
`

Reply:
Well explained!